hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/RocksDBStoreMetrics.java

```diff
@@ -105,6 +105,8 @@ public class RocksDBStoreMetrics implements MetricsSource {
   private static final String NUM_FILES_AT_LEVEL = "num_files_at_level";
   private static final String SIZE_AT_LEVEL = "size_at_level";
 
+  private static final String LAST_SEQUENCE_NUMBER = "last_sequence_number";
```

Comment:
Can we also expose wal_ttl in metrics and wal_size ?

Reply:
@devmadhuu ， wal_ttl and wal_size value are statically configured value. They can be retrieved from the rocksdb MANIFEST file.